### PR TITLE
Add Code tab to Radio Group page

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+import uncontrolled from "./examples/uncontrolled";
+import errorHandling from "./examples/errorHandling";
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        {" "}
+        <tr>
```
Comment (raquelarrojo) [inline website/screens/components/radio-group/code/RadioGroupCodePage.tsx:18]: Remove this white space.

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    subSections: [
+      {
+        title: "Basic usage",
+        content: (
+          <>
+            <Example example={basicUsage} defaultIsVisible />
+          </>
+        ),
+      },
+      {
```
Comment (raquelarrojo) [inline website/screens/components/radio-group/code/RadioGroupCodePage.tsx:177]: Basic usage example is not needed here, as basic usage should be controlled and it is the same as the controlled example here.

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        title: "Controlled",
+        content: (
+          <>
+            <Example example={controlled} defaultIsVisible />
+          </>
+        ),
+      },
```
Comment (raquelarrojo) [inline website/screens/components/radio-group/code/RadioGroupCodePage.tsx:184]: Please remove the React Fragment here `<></>`.

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+      {
+        title: "Uncontrolled",
+        content: (
+          <>
+            <Example example={uncontrolled} defaultIsVisible />
+          </>
+        ),
```
Comment (raquelarrojo) [inline website/screens/components/radio-group/code/RadioGroupCodePage.tsx:191]: Please remove the React Fragment here `<></>`.

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx (hunk 4 above)
Comment (Lisa159) [reply website/screens/components/radio-group/code/RadioGroupCodePage.tsx]: Done

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx (hunk 2 above)
Comment (Lisa159) [reply website/screens/components/radio-group/code/RadioGroupCodePage.tsx]: Done

File: website/screens/components/radio-group/code/RadioGroupCodePage.tsx (hunk 1 above)
Comment (Lisa159) [reply website/screens/components/radio-group/code/RadioGroupCodePage.tsx:16]: Done

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**